src/docs/background/10-custom-pricing/index.md

```diff
@@ -0,0 +1,263 @@
+  $100,
+  and after that each additional person costs $20 extra.
+
+All of these can be achieved using _custom pricing_. Custom pricing
+provides the marketplace website or app with a wide range of ways to
+affect how bookings are created. In this article we'll look at bit
+closer on how pricing works in Flex and how it can be extended with
+custom pricing.
+
+## How pricing works in Flex
+
+In Flex, the total price of a transaction is defined by its _line
+items_. Line items describe what is included in a transaction. It
+depends on the transaction process what the line items contain but it
+can be a varying set of things from number of booked units to customer
+and provider commissions and payment refunds. The line items of a
+transaction depend on the _actions_ that are used in the transaction
+process in the transition(s) in which the transaction is created. In
+general, actions can perform anything that is related to the transaction
+process like make payments and send notifications. But when it comes to
+pricing, it's up to the actions to construct the right set of line items
+based on the request parameters that are passed to the API.
+
+Different actions can be used to create different pricing schemes. A
+night-based pricing action calculates the number of nights between the
+start and end of the booking to calculate the number of booked units
+whereas a day-based action respectively calculates the number of days. A
+unit-based action just takes a _quantity_ parameter that it uses to
+construct the line item that holds the information about the number of
+booked units.
+
+Every line item has a unit price and one of the following attributes:
+_quantity_ and _percentage_. The quantity attribute can be used to
+denote the number of booked units, like number of booked nights. The
+percentage param is used when modeling commissions for example. Based on
+these attributes a line total is calculated for each line item. Line
+totals then define the total payin and payout sums of the transaction.
+
+When using a night, day and unit based pricing actions the Flex API
+constructs the line items based on parameters passed in a request. The
+marketplace website or app does not need to concern how the line items
+are created but it can rely on the information they contain when
+presenting the transaction information in the UI.
+
+## More complex pricing schemes with custom pricing
+
+In addition to a pricing scheme that is based on booking length or
+number of booked units, pricing can be extended or completely
+re-designed by using _custom pricing_. Custom pricing works so that it
+uses a specific custom pricing action called `set-line-items-and-total`
+in the transaction process. As the name suggests, this action allows you
+to define any number of line items you need. So the client application,
+that is your marketplace website or application, passes the booking
+related line items to the API when initiating a transaction. When
```

Comment:
Pricing is not restricted to the initial transition but that's typically where it happens. Maybe more a FYI than change request; explaining it like this might be easier to understand even though not fully technically accurate.